configs/env_config.py

```diff
@@ -5,4 +5,17 @@
 
 class EnvConfig(BaseConfig):
     def _get_config_value(self, variable_name):
-        return os.getenv(variable_name)
+        if variable_name in (
+            "AWS_REGION",
+            "FLASKS3_ACTIVE",
```

Comment:
FLASKS3_ACTIVE needs to be set to False if we dont want to use it

Reply:
FLASKS3_ACTIVE has been set in .env file locally, removed logic which is no longer required